assets/pages/new-contributions-landing/contributionsLandingActions.js

```diff
@@ -377,16 +383,24 @@ const paymentAuthorisationHandlers: PaymentMatrix<(Dispatch<Action>, State, Paym
     Stripe: (dispatch: Dispatch<Action>, state: State, paymentAuthorisation: PaymentAuthorisation): void => {
       dispatch(executeStripeOneOffPayment(stripeChargeDataFromAuthorisation(paymentAuthorisation, state)));
     },
-    DirectDebit: () => { logInvalidCombination('ONE_OFF', 'DirectDebit'); },
-    None: () => { logInvalidCombination('ONE_OFF', 'None'); },
+    DirectDebit: () => {
+      logInvalidCombination('ONE_OFF', 'DirectDebit');
```

Comment:
You may want to store 'ONE_OFF', 'ANNUAL' and 'MONTHLY' as constants somewhere given they are used across files.

Reply:
I think the reason we don't do this is because we use flow types to safeguard against typos. It's a convention we use across the entire codebase but I'd be interested in peoples opinions @joelochlann @Ap0c ?

Anyway, this change isn't even really part of this PR, it was just some auto formatting intelliJ did and I thought I'd keep in as I preferred it!

Reply:
Flow will guard against this, yes. Storing these strings in an object might make autocomplete slightly easier but not too much in it